receiver/splunkenterprisereceiver/README.md

```diff
@@ -23,16 +23,38 @@ Example:
 
 ```yaml
 extensions:
-    basicauth/client:
+    basicauth/indexer:
+        client_auth:
+            username: admin
+            password: securityFirst
+    basicauth/cluster_master:
         client_auth:
             username: admin
             password: securityFirst
 
 receivers:
     splunkenterprise:
-        auth: basicauth/client
-        endpoint: "https://localhost:8089"
-        timeout: 45s
+        indexer:
+            auth: 
+              authenticator: basicauth/indexer
+            endpoint: "https://localhost:8089"
+            timeout: 45s
+        cluster_master:
+            auth: 
+              authenticator: basicauth/cluster_master
+            endpoint: "https://localhost:8089"
+            timeout: 45s
+
+exporters:
+  logging:
+    loglevel: info
+
+service:
+  extensions: [basicauth/indexer, basicauth/cluster_master]
```

Comment:
Thank you for documenting this! I can see some getting hung up on the multiple `basicauth` entries until the concept clicks.